packages/formation/sass/modules/_m-back-to-top.scss

```diff
@@ -0,0 +1,73 @@
+//=============================
+// Up To Top Button Styles
+//=============================
+#top-button-container {
+  bottom: 0;
+  pointer-events: none;
+  position: fixed;
+  width: 100%;
+
+  &.va-top-button-container-relative {
```

Comment:
~How and why is this used? (Should it be documented in Storybook?)~
Ahhhhh, I see. It's to toggle the "stickiness" when the footer is scrolled into view. Maybe make a comment to that effect here?